libs/shared/console-shared/src/lib/deploy-other-commit-modal/ui/deploy-other-commit-modal.tsx

```diff
@@ -0,0 +1,146 @@
+                  <div className="flex flex-col rounded-md border border-element-light-lighter-500">
+                    {commitsByDay[date].map((commit, index) => (
+                      <div
+                        data-testid="commit-box"
+                        key={commit.git_commit_id}
+                        onClick={() => setSelectedCommitId(commit.git_commit_id)}
+                        className={`h-[5.5rem] flex items-center transition-all justify-between cursor-pointer px-5 border-element-light-lighter-500 ${
+                          currentCommitId !== commit.git_commit_id && selectedCommitId === commit.git_commit_id
+                            ? 'bg-brand-50 outline-brand-500 outline'
+                            : ''
+                        } ${commitsByDay[date].length - 1 === index ? '' : 'border-b'}
+                    ${currentCommitId === commit.git_commit_id ? 'bg-green-50' : ''}
+                    ${index === 0 ? 'rounded-t-md' : ''} ${
+                          index === commitsByDay[date].length - 1 ? 'rounded-b-md' : ''
+                        }
+                    `}
+                      >
+                        <div className="w-full">
+                          <div className="flex justify-between w-full">
+                            <p className="text-text-500 font-medium mb-1.5">
+                              <a href={commit.commit_page_url} target="_blank" className="hover:text-brand-500">
+                                <Truncate truncateLimit={45} text={commit.message} />
+                              </a>
+                            </p>
+                            <TagCommit commitId={commit.git_commit_id} />
+                          </div>
+
+                          <div className="flex justify-between w-full">
+                            <p className="text-text-400">committed {timeAgo(new Date(commit.created_at))} ago</p>
+                            <p>
+                              {currentCommitId !== commit.git_commit_id &&
+                                selectedCommitId === commit.git_commit_id && (
+                                  <span className="text-brand-500 text-ssm font-medium">Selected version</span>
+                                )}
+                              {currentCommitId === commit.git_commit_id && (
+                                <span className="text-green-500 text-ssm font-medium">Current version</span>
+                              )}
+                            </p>
+                          </div>
+                        </div>
+                      </div>
+                    ))}
+                  </div>
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+      )}
+
+      {!props.isLoading && Object.keys(commitsByDay).length === 0 && (
+        <div className="flex justify-center">
+          <p className="text-text-400 text-sm">No commits found</p>
```

Comment:
For this, I think you can use the same design like another empty placeholder for search (you can find it on the Breadcrumb and for InputSelect)

<img width="514" alt="Capture d’écran 2022-11-15 à 14 18 44" src="https://user-images.githubusercontent.com/533928/201929397-4e8555e9-4d02-4b3b-8b50-ceb3b55bbafb.png">